modules/auxiliary/admin/kerberos/forge_ticket.rb

```diff
@@ -34,7 +34,7 @@ def initialize(info = {})
           'Stability' => [CRASH_SAFE],
           'SideEffects' => [IOC_IN_LOGS],
           'Reliability' => [],
-          'AKA' => ['Silver Ticket', 'Golden Ticket', 'diamond', 'sapphire', 'Ticketer', 'Klist']
```

Comment:
These previous values were just SEO hackery; Now actions are searchable this can be trimmed down